src/ResourceManager/RecoveryServices.Backup/CmdletParameterHelpMessages.cs

```diff
@@ -33,6 +33,16 @@ internal static class Container
         internal static class Common
         {
             public const string Vault = "The Azure Backup vault object which is the parent resource.";
+            public const string WorkloadType = "Workload type of the resource (for example: AzureVM, WindowsServer).";
```

Comment:
Remove WindowsServer from WorkloadType. WindowsServer should fall under ContainerType